code-challenges/CC5_string_methods/challenges-05.test.js

```diff
@@ -74,8 +74,14 @@ const gruffaloCrumble = {
   ]
 };
 
-const splitFoods = (recipe) => {
-  // solution code here
+const listFoods = (recipe) => {
+  const result = [];
+  recipe.ingredients.forEach(item=> {
+    let withoutAmount = item.slice(recipe.indexOf('')+1);
```

Comment:
recipe (the original entire object passed to the splitFoods function) does not seem to be the variable you want to try the indexOf method on. If you are trying to locate a space inside of a string, you want to have a space between the quotes inside the indexOf method